src/main/java/edu/kit/ipd/dbis/controller/GraphEditorController.java

```diff
@@ -167,9 +160,8 @@ public void addNewGraph(PropertyGraph<Integer, Integer> graph) throws InvalidGra
 	 *
 	 * @param graph the PropertyGraph<V,E> to check.
 	 * @return true if the given graph is valid.
-	 * @throws InvalidGraphInputException the invalid graph input exception
 	 */
-	public Boolean isValidGraph(PropertyGraph<Integer, Integer> graph) throws InvalidGraphInputException {
+	public Boolean isValidGraph(PropertyGraph<Integer, Integer> graph) {
```

Comment:
why does this method return a boolean Object?

Reply:
Because it checks wether the graph is valid or not.

Reply:
But the method throws also an InvalidGraphInputException (i think if the graph is not valid) so in the case the method return something the boolean is always true because the graphInput is valid.

Reply:
@AlexanderSchorn it should return a boolean value, which is a primitive data type. Not the object.